packages/simcore-sdk/tests/unit/test_node_ports_v2_port.py

```diff
@@ -74,21 +77,66 @@ def e_tag() -> str:
     return "1212132546546321-1"
 
 
-##################### FIXTURES
+@contextmanager
+def symlink_to_file_with_data(file_name: Path) -> Iterator[Path]:
```

Comment:
I do not understand why you use a contextmanager here instead of a fixture? what does that bring?

Reply:
I managed to refactor it using fixtures